server/controllers/index.js

```diff
@@ -58,25 +58,38 @@ router.get('/auth', (req, res) => {
     }
 });
 
+/* Log in the user */
+router.post('/auth/signup', (req, res) => {
+    const { username, password } = req.body;    
+    db.createUser(username, password)
+    .then(user => {
+        const real = true;
+        const id = user.dataValues.id;
+        req.session.user = { username, id, real };
+        req.session.roomId = null;
+        currentUsers.push({ username });
+        res.sendStatus(200);
+    }).catch(err => {
+        console.log(err);
```

Comment:
`console.error()` is preferred, so you can filter by verbosity. you may also find that you enjoy better support in bash (i.e. errors sent to a separate file descriptor, stderr, which is not fed into pipes)